File: tinypng/compress.php
<?php
require_once("vendor/autoload.php");
Tinify\setKey(getenv("TINY_KEY"));

echo "COMPRESSION BY URL\n";
$tiny = Tinify\fromFile("images/in/small.png");
$tiny->toFile("images/out/small-compressed.png");
echo "Compression done, thank you come again!\n";
echo "Count: " . Tinify\getCompressionCount() . "\n";

echo "COMPRESSION BY UPLOAD\n";
$tiny = Tinify\fromUrl("https://raw.githubusercontent.com/tinify/tinify-ruby/master/test/examples/voormedia.png");
$tiny->toFile("images/out/small-compressed-url.png");
echo "Compression done, thank you come again!\n";
echo "Count: " . Tinify\getCompressionCount() . "\n";

?>

File: tinypng/compress.py
# pip install tinify
# -*- coding: utf-8 -*-
import os, tinify, sys

tinify.key = os.getenv('TINY_KEY')
print tinify.key

tinify.from_file(u'images/in/你好.jpg').to_file('images/out/small-compressed.png')
print "Compression Done! Compression done this month: {0}".format(tinify.compression_count)

tiny = tinify.from_url('https://raw.githubusercontent.com/tinify/tinify-ruby/master/test/examples/voormedia.png')
tiny.to_file('images/out/small-compressed.png')
print "Compression Done! Compression done this month: {0}".format(tinify.compression_count)

# http://stackoverflow.com/questions/9644099/python-ascii-codec-cant-decode-byte